airgun/views/contentview.py

```diff
@@ -192,6 +256,58 @@ def is_displayed(self):
         )
 
 
+class ContentViewVersionDetailsView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and len(self.breadcrumb.locations) > 3
+                and self.breadcrumb.locations[0] == 'Content Views'
+                and self.breadcrumb.locations[2] == 'Versions'
+        )
+
+    @View.nested
+    class yum_repositories(SatSecondaryTab, SearchableViewMixin):
+        TAB_NAME = 'Yum Repositories'
+        table = SatTable('.//table')
+
+    @View.nested
+    class rpm_packages(SatSecondaryTab):
```

Comment:
why not inherit from SearchableViewMixin also?

Reply:
if i override is_searchable and search, then what's the point of inheritance here if 100% of features will be overwritten? :)
btw is_searchable doesn't really make sense here as welcome message might be shown only on entities list page, not inside some entity details tab.